chicken_app/src/Pages/Profile/ConfirmEmail.js

```diff
@@ -1,9 +1,57 @@
-import React from 'react'
+import { React, useState, useRef } from "react";
+import MailIcon from "@mui/icons-material/Mail";
+import Alert from "@mui/material/Alert";
+import AlertTitle from "@mui/material/AlertTitle";
+import { getAuth, sendEmailVerification } from "firebase/auth";
+
+import { Box, ThemeProvider, createTheme, Typography } from "@mui/material";
+import { borders } from "@mui/system";
+import { Button } from "@mui/material";
+
+const theme = createTheme({
+  palette: {
+    background: {
+      paper: "#f1faf8",
+    },
+    typography: {
+      body1: {
+        color: "#6bb09e",
+      },
+    },
+    text: {
+      primary: "#6bb09e",
+    },
+  },
+});
 
 function ConfirmEmail() {
+  const [emailSent, setEmailSent] = useState({ isEmailSent:true, emailSentDetails:"Email verification sent!"});
+  const auth = getAuth();
+  console.log("emailSent", emailSent);
+  sendEmailVerification(auth.currentUser).then(() => {
+    setEmailSent({ isEmailSent:true, emailSentDetails:"Email verification sent!"});
+  });
   return (
-    <div>ConfirmEmail</div>
-  )
+    <ThemeProvider theme={theme}>
+      <Box align='flex' mx={{ sm:'auto', xs:'24px' }} sx={{maxWidth:'700px', mt:{ sm:'116px', xs:'24px'}, borderColor: 'text.primary', color: "success.main" }}>
+        <Alert icon={<MailIcon fontSize="inherit" sx={{marginTop: "3px", color: '#6bb09e'}}/>} severity="success" sx={{backgroundColor: "background.paper", marginTop: "15px", marginBottom: "-5px", }}>
+          <AlertTitle variant="h5" sx={{color: '#6bb09e', fontColor: "text.primary", fontWeight: "bold" }}>
```

Comment:
Take a look at Components/Theme.js, most of the styling for the typography is already set, 
I add a typography component inside the AlertTtitle rather than adding text styling to the AlertTitle

For textcolor though good idea thinking to add that to Alert as it's used fully in there :D